tasks/app/site_settings.py

```diff
@@ -45,7 +45,10 @@ def get_value(context, key, default=None):
 
 
 @app_context_task()
-def get_public_data(context):
+def get_public_data(context, debug=False):
+
+    if debug:
```

Comment:
do we want to push this debug option to main?

Reply:
Absolutely.
In a pyinvoke task this is extremely useful.
It allows a user (who is logged into the server) to be able to  really interrogate the system to diagnose what is actually happening.